docs/Delivery-Practices/Build-and-Release/Engineering/Peer-Reviewing/Release-Plan-Review.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Release Plan Review
+authors: 
+  - Rhodri Hewitson
+reviewed: 
+reviewer:
+next*review: 2023-06-01
+---
+# Release Plan Review
+
+## Why do we have release documentation? 
+Until we have fully automated our release process, as engineers we must fill out a release plan to make sure care and consideration is taken to ensure we reduce downtime as much as possible and release the BLI as safely as possible,
+
+## Who can approve a release? 
+All senior developers and tech leads can review release documentation, but an engineer cannot review a release document they've had a hand in writing. 
```

Comment:
Presumably this should be "Senior Software Engineer"?

And then in the scond part, "but an SSE cannot review a release document they've had a hand in writing"?

Reply:
Oh my good spot re the use of developer. Re the had a hand in writing i think that's probably true for anyone regardless of role?

Reply:
> Re the had a hand in writing i think that's probably true for anyone regardless of role?

Yes! Fair point.